build/tasks.ts

```diff
@@ -260,4 +260,30 @@ const ensurePatched = register("vscode:patch", async (runner) => {
 	}
 });
 
+register("package", async (runner, releaseTag) => {
+	if (!releaseTag) {
+		throw new Error("Please specify the release tag.");
+	}
+
+	const releasePath = path.resolve(__dirname, "../release");
+
+	const archiveName = `code-server-${releaseTag}-${os.platform()}-${os.arch()}`;
+	const archiveDir = path.join(releasePath, archiveName);
+	fse.removeSync(archiveDir);
+	fse.mkdirpSync(archiveDir);
+
+	const binaryPath = path.join(__dirname, `../packages/server/cli-${os.platform()}-${os.arch()}`);
+	const binaryDestination = path.join(archiveDir, "code-server");
+	fse.copySync(binaryPath, binaryDestination);
+	fs.chmodSync(binaryDestination, "755");
+	["README.md", "LICENSE"].forEach((fileName) => {
+		fse.copySync(path.resolve(__dirname, `../${fileName}`), path.join(archiveDir, fileName));
+	});
+
+	runner.cwd = releasePath;
+	await os.platform() === "linux"
+		? runner.execute("tar", ["-cvzf", `${archiveName}.tar.gz`, `${archiveName}`])
+		: runner.execute("zip", ["-r", `${archiveName}.zip`, `${archiveName}`]);
```

Comment:
For our use case, is tar preferable to zip? And if so, is there any way for us to set a conditional based on whether or not said commands are installed rather than OS platform?

Reply:
The goal is to provide tarballs for Linux and zips for Mac & Windows. This runs in the CI and is what will be available for download on the "releases" page.